docs/intro-to-lora.md

```diff
@@ -141,6 +141,31 @@ Now that we have set up the gateways and they can reach the internet, it's time
 
 <span class="tips">**Tip:** LORIOT has a 'Gateway Tap', which you can access from your gateway page. This allows you to see what messages are being picked up by the gateway - very useful when debugging.</span>
 
+### Automatically starting the LORIOT binary when the gateway starts
+
+#### Kerlink IoT station
+
+If you follow the installation steps in the LORIOT documentation, the binary will automatically start whenever the gateway boots. No additional steps required.
+
+#### Multitech Conduit
+
+1. Log in over SSH or via the serial port.
+1. Create a new file ``/etc/init.d/start-loriot.sh``. 
+1. Edit this file (f.e. via `nano /etc/init.d/start-loriot.sh`) and add the following content:
```

Comment:
f.e. is foreign to 90% of your readers (obviously I'm making the number up, but the point stands)